examples/connectivity/plot_canica_resting_state.py

```diff
@@ -46,11 +46,10 @@
 import nibabel
```

Comment:
FWIW `flake8 examples | grep nibabel` is probably better since it is going to give you PEP8 warnings as well.